### ISSUE TITLE
Installation aborted --Wordpress 6.1.0 desktop app for Windows 10 

### ISSUE THREAD

**Issue by pkeiiysrs** [Automattic/wp-desktop#962]
#### Steps to reproduce
1.I did a download of the installation exe file from Wordpress.com, file is stored in C:\windows\USER\downloads\wordpress.com-win32-setup-6.1.0.exe. The installer runs and I click the INSTALL button.
2.Tried installing for any user and also for a single user, both fail with the message "Installation Aborted, the installation was not completed."  Screenshot is attached to this message.
3.The installer sits there, spinning. I hit CANCEL to close it.
4.In case the downloaded file was corrupted, I repeated the download and got the same results.

#### What I expected
Expected the installer to run to completion.

#### What happened instead
Installation is aborted.

#### App Version / OS version
Wordpress 6.1.0    Windows 10 version 1909, Installed 11/18/2019  OS BUILD: 18363.1082

#### Screenshot / Video
![image](https://user-images.githubusercontent.com/34556541/95793741-19f6c900-0c9b-11eb-81c0-f5dd924b71c5.png)

**Comment by designsimply** [comment]
I tested the WordPress.com Desktop app 6.1.0 installer on Windows 10 and it worked in my test.

Video: [2m31s](https://cloudup.com/cZqMZhjmp7f)
<sup>Tested with wordpress.com-win32-setup-6.1.0.exe on Windows 10.</sup>

@pkeiiysrs may I confirm with you that you downloaded the app directly from https://apps.wordpress.com/desktop/ and may I also ask if you would try completely uninstalling and reinstalling the app as a next troubleshooting step?

**Comment by pkeiiysrs** [comment]
Just to be sure, I used the URL that you provided, and I downloaded WP6.1.0
twice,  once with MS-Edge and once with Google Chrome.
I tried to do the install, and I got the same result with both browsers.
Both fail with the message  "Installation Aborted".
If it's not the browsers, then could it be some dll that needs fixing or a
registry entry that needs to be reset?
FYI. I am using Norton Anti-Virus, which pops up when I do the download. It
tells me that the exe is safe.

Patrick Kelly
www.NEW-TALES.com <http://www.new-tales.com>

**Comment by designsimply** [comment]
@pkeiiysrs can you please try completely uninstalling the WordPress.com Desktop app, make sure it has been removed, and then try installing it again?

**Comment by pkeiiysrs** [comment]
Hi Sheri,
Wordpress is not listed in the menu of applications, however I went to
Windows Settings and did a search and Wordpress 4.6.0  shows up as
installed. However when I try to uninstall it then I get an error message
that the system cannot find the Uninstall program.

[image: image.png]

  I found  this directory C:\Users\abang\AppData\Roaming\Wordpress.com
The screenshot below shows the contents of the Wordpress.com directory.

[image: image.png]

Does that help?

Patrick Kelly
www.NEW-TALES.com <http://www.new-tales.com>

**Comment by designsimply** [comment]
@pkeiiysrs I'm sorry but your images did not come through. It sounds like more troubleshooting is needed as this may not be a bug but rather a problem with your system setup. Can you please email mobile-support@automattic.com directly and send them a link to this issue on GitHub https://github.com/Automattic/wp-desktop/issues/962 and ask them for help with troubleshooting? They will do their best to help you. If you work with the support team and are able to determine that there really is a bug, including providing a short set of steps to reproduce the problem, then we can re-open this case on GitHub so it can get attention but first you should make sure to troubleshoot and document the problem together with the support team.

**Comment by pkeiiysrs** [comment]
Ok, thanks.
Patrick Kelly
www.NEW-TALES.com <http://www.new-tales.com>

**Comment by designsimply** [comment]
Thanks back at you! I'm keen to get you the help you need. I think working with the support team is the best place to continue debugging in the short term, and I also recommend doing a little digging to figure out how to uninstall the app because I bet it will start working properly after a fresh install in your case.

**Comment by pkeiiysrs** [comment]
RE: Github issue #962
Sheri,
You were correct. It's WORKING !
I found an old version 4.6.0 not installed but the name was stuck in the
list of apps. It did not show up on the normal Windows App listing. It
showed up when I went to Settings and  searched for 'word'. It still would
not de-install. Then I went to the old Control Panel and had to delete it
there, too. Then I went back to settings and was able to delete it.

Patrick Kelly
www.NEW-TALES.com <http://www.new-tales.com>
